Tests/UnitTestsHostApp/SceneDelegate.swift

```diff
@@ -1,60 +0,0 @@
-//
-//  Copyright RevenueCat Inc. All Rights Reserved.
-//
-//  Licensed under the MIT License (the "License");
-//  you may not use this file except in compliance with the License.
-//  You may obtain a copy of the License at
-//
-//      https://opensource.org/licenses/MIT
-//
-//  SceneDelegate.swift
-//
-//  Created by Andrés Boedo on 9/24/21.
-
-import UIKit
-
-class SceneDelegate: UIResponder, UIWindowSceneDelegate {
```

Comment:
This wasn't needed in the host app (and it's iOS 14.0+ only, so I just removed it).